tutorials/sdf_to_usd_arms.md

```diff
@@ -0,0 +1,111 @@
+# Manipulating a SDF Panda Robot Arm in Isaac Sim with USD
+
+Download the panda model from Fuel (make sure you have [Ignition Fuel Tools installed](https://ignitionrobotics.org/api/fuel_tools/5.0/install.html))
+
+```bash
+ign fuel download -u "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/panda with ignition position controller model"
+```
+
+Copy the empty world [`empty.sdf`](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo6/examples/worlds/empty.sdf) to a file called `panda.sdf` and add these lines inside the `<world>` tag:
+
+```xml
+<include>
+  <pose>0 0 0 0 0 0</pose>
+  <name>panda</name>
+  <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/panda with ignition position controller model</uri>
+</include>
+```
+
+Run the converter (you must build sdformat first):
+
+```bash
+usdConverter panda.sdf panda.usd
+```
+
+Open IsaacSim and load the `panda.usd` file.
+
+Add to the `panda` prim the following:
+
+ - Create -> Isaac -> ROS -> Clock
+ - Create -> Isaac -> ROS -> Joint State
+    - Set *targetPrims* to `/fuel/panda`
+ - Create -> Isaac -> ROS -> Pose tree
+    - Set *articulationPrim* to `/fuel/panda`
```

Comment:
should these paths be `/empty/panda` instead of `/fuel/panda`? The `empty.sdf` file that should have been copied above has a world name of `empty`, and the `usdconverter` sets the usd root name to the sdf world name: https://github.com/ignitionrobotics/ign-gazebo/blob/555a84ea54a6f92b46929eb2f1a43daa13b1fadb/examples/worlds/empty.sdf#L63